### ISSUE TITLE
Creating a list with all personal bests

### ISSUE THREAD

**Issue by LinusFresz** [suushiemaniac/PrisMakeAWish#1]
I would like to request a feature with the following attributes:
The feature should be able to show all personal bests you have made in the different events in Prisma. 
This could be look like:
| Event     | Single    | mo3   | avg5  | avg12     | avg50     | avg100    | global average    |
|-------    |--------   |------ |------ |-------    |-------    |--------   |----------------   |
| 2x2x2     | 1         | 1.5   | 2     | 2.5       | ...       |           |                   |
| 3x3x3     | 10        | 10.5  | 11    | ...       |           |           |                   |
| ...       |           |       |       |           |           |           |                   |

(well.. maybe a better alignment for the columns)
So basically just a table with all your pbs.

**Comment by gregorbg** [comment]
Is this something you might be looking for?
https://github.com/suushiemaniac/PrisMakeAWish/commit/d58f93b5a7a9d95afe0ae1a84ca18b6c6b0381f2
